src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -2172,16 +2275,52 @@
           short-summary: Name or ID of the VM to target.
         - name: --filters
           short-summary: JSON encoded list of packet filters. Use `@<file path>` to load from file.
+    examples:
+        - name: Create a packet capture session on a VM
+          text: az network watcher packet-capture create -g MyRg -n packetCaptureName --vm MyVm --storage-account gwteststorage123abc
+        - name: Create a packet capture session on a VM with optional filters
```

Comment:
Improve description, describe the actual parameters provided. Should use the YAML text block `>` operator to break up across multiple lines.